apps/modernization-ui/src/apps/page-builder/pages/BusinessRulesLibrary/useBusinessRulesAPI.ts

```diff
@@ -30,11 +30,12 @@ export const fetchBusinessRules = (
         return PageRuleControllerService.findPageRuleUsingPost({
             authorization,
             request,
+            pageId: pageId.toString(),
             page: currentPage && currentPage > 1 ? currentPage - 1 : 0,
             size: pageSize,
             sort
         })
-            .then((response: any) => {
+            .then((response: Page_ViewRuleResponse_) => {
                 return response || [];
             })
             .catch((error: any) => {
```

Comment:
Question: is there a way to give the a type instead of any?

Reply:
Its set to Page_ViewRuleResponse_